xrpl/core/binarycodec/types/issued_currency.py

```diff
@@ -0,0 +1,90 @@
+"""Codec for serializing and deserializing issued currency fields."""
+
+from __future__ import annotations
+
+from typing import Any, Dict, Optional, Type, Union
+
+from xrpl.core.binarycodec.binary_wrappers.binary_parser import BinaryParser
+from xrpl.core.binarycodec.exceptions import XRPLBinaryCodecException
+from xrpl.core.binarycodec.types.account_id import AccountID
+from xrpl.core.binarycodec.types.currency import Currency
+from xrpl.core.binarycodec.types.serialized_type import SerializedType
+from xrpl.models.currencies import IssuedCurrency as IssuedCurrencyModel
+
+
+class IssuedCurrency(SerializedType):
```

Comment:
Is this type not covered by the Amount type?  https://github.com/XRPLF/xrpl-py/blob/master/xrpl/core/binarycodec/types/amount.py

Reply:
Amounts have an amount. This is just the currency.